source/common/router/retry_state_impl.cc

```diff
@@ -76,6 +76,7 @@ RetryStateImpl::RetryStateImpl(const RetryPolicy& route_policy, Http::HeaderMap&
 RetryStateImpl::~RetryStateImpl() { resetRetry(); }
 
 void RetryStateImpl::enableBackoffTimer() {
+  // TODO(ramaraochavali): Implement JitteredExponentialBackOff and refactor this.
```

Comment:
Ah, I see, you have thought about jitter here..

Reply:
+1, and, honestly, you should be using jittered backoff for the management connection also. I would not support any backoff policy that is not jittered...

Reply:
@mattklein123 i am ok with that. Let us review the PR for jittered backoff https://github.com/envoyproxy/envoy/pull/3791 and I can move management connection to use this. @htuch thoughts?

Reply:
Yeah, I think have jitter as default (and only built-in) scheme makes sense, best not to provide sharp objects that aren't needed.